ui/app/AppLayouts/Wallet/controls/EditSlippagePanel.qml

```diff
@@ -0,0 +1,113 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Popups 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+
+import shared.controls 1.0
+
+import utils 1.0
+
+Control {
+    id: root
+
+    property bool loading
+    property var selectedToToken
```

Comment:
Pls change it so that we're passing just the simple properties around (symbol, image, ...)

Reply:
I dont understand why it is needed here. we need various properties from the selectedToToken such as symbol and image and maybe something else in future? is there a reason why we should break it down at this level?